### PR TITLE
feat(endpoint): update count distinct

### PR DESCRIPTION
**Pull request by OlegDO**

### CODE REVIEW

**Review by MatthewPattell — CHANGES_REQUESTED** [review]
rewrite

File: src/services/endpoint.ts [hunk 1]
```diff
@@ -525,7 +525,7 @@ const hasEmptyCondition = <TEntity>(query: SelectQueryBuilder<TEntity>): boolean
   }
 
   // condition should contain at least one parameter or equal (number | string)
-  return !/([a-z\s."]+)([!=<>]+|like|is|in)\s?(:|\(:|[0-9]|')/i.test(condition);
+  return !/([a-z\s."]+)([!=<>]+|like|is|in)\s?(:|\(:| [0-9]|')/i.test(condition);
```
Comment (MatthewPattell) [inline src/services/endpoint.ts]: you add space? use `\s`

**Review by MatthewPattell — CHANGES_REQUESTED** [review]

File: src/services/endpoint.ts (hunk 1 above)
Comment (OlegDO) [reply src/services/endpoint.ts]: I am not the one who did it. Seems strange 
image

Comment (OlegDO) [reply src/services/endpoint.ts]: I'll revent "no space"

**Review by MatthewPattell — APPROVED** [review]

**Pull request merged**